CppExport/src/dependency_analysis/DependencyUnit.cpp

```diff
@@ -30,6 +30,39 @@
 
 namespace CppExport {
 
-DependencyUnit::DependencyUnit(QString name, Model::Node* node) : name_(name), node_(node) {}
+DependencyUnit::DependencyUnit(QString name, Model::Node* node) : name_(name), node_(node)
+{
+	targets_ = calculateTargets(node);
```

Comment:
target is a field, so it should be initialized like all other fileds: ` name_{..}, ...., targets_{calculateTargets(node)}`